server/src/assetComputeDevTool.js

```diff
@@ -139,6 +139,11 @@ class AssetComputeDevTool {
         );
     }
 
+    async isJournalReady() {
+        const isReady = await this.assetCompute.isEventJournalReady();
```

Comment:
You can simplify this:
```suggestion
        return this.assetCompute.isEventJournalReady();
```